### DISCUSSION TITLE
Use speakers and displays for presence detection

### DISCUSSION THREAD

**Discussion by dnestico** [leikoilja/ha-google-home#611] [General]
Recently Google rolled out the ability to use the both nest speakers and displays as part of it's presence detection.

Can this be taken advantage of and added to this addon? Not a must but fs would be cool.

![Screenshot_20220924-142124.png](https://user-images.githubusercontent.com/3034555/192113113-7652b864-64ba-4305-81d9-47657c06fe9e.png)

**Comment by leikoilja** [comment]
I created a new feature request in #614 and answered there. Unfortunately, it's not possible with the existing google APIs that we have to work with.
